build/common.props

```
@@ -3,7 +3,7 @@
     <!-- Extensions can have independent versions and only increment when released -->
     <Version>3.0.0$(VersionSuffix)</Version>
     <ExtensionsVersion>5.0.0$(VersionSuffix)</ExtensionsVersion> <!-- WebJobs.Extensions -->
-    <CosmosDBVersion>4.8.0$(VersionSuffix)</CosmosDBVersion>
+    <CosmosDBVersion>4.9.0$(VersionSuffix)</CosmosDBVersion>
```

```suggestion
    <CosmosDBVersion>4.8.1$(VersionSuffix)</CosmosDBVersion>
```

Use a patch version. Minor version is used when dependencies or APIs change